client/src/app/app.routes.ts

```diff
@@ -10,6 +10,8 @@ import { ProvincialRankingsComponent } from './pages/provincial-rankings/provinc
 import { ProvincialRecordsComponent } from './pages/provincial-records/provincial-records.component';
 import { PrivacyComponent } from './pages/privacy/privacy.component';
 import { TermsComponent } from './pages/terms/terms.component';
+import { VolunteerComponent } from './pages/volunteer/volunteer.component';
+import { faqsComponent } from './pages/faqs/faqs.component';
```

Comment:
From a naming convention point of view, this should start with a capital. Please change `faqsComponent` to `FAQComponent`.

Reply:
Done, and moving on.

Reply:
This is finished.